### PR TITLE
init linear simulator as module 

### PR DESCRIPTION
**Pull request by jedyeo**
linear_simulator needs to be recognized as a module so reconstructSPI can successfully import from this directory.

See https://github.com/compSPI/reconstructSPI/pull/48

### CODE REVIEW

**Comment by jedyeo** [comment]
@geoffwoollard  do you know why it ignored the ignore check

**Comment by geoffwoollard** [comment]
> @geoffwoollard do you know why it ignored the exclude check

Where is the exclude check specified? Could something be added to here: https://github.com/compSPI/simSPI/blob/master/.github/workflows/lint.yml ?

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -39,4 +39,4 @@ repos:
       additional_dependencies:
         - flake8-docstrings
         - flake8-import-order
-      exclude: simSPI/__init__.py,tests/__init__.py
+      exclude: simSPI/__init__.py,tests/__init__.py,simSPI/linear_simulator/__init__.py
```
Comment (geoffwoollard) [inline .pre-commit-config.yaml:42]: this is the precommit, for when you are comiting locally and you get the automatic checks or changes (e.g. with black).

This is different from how the linting and testing run on github workflows.

Perhaps add something in here: https://github.com/compSPI/simSPI/blob/master/.github/workflows/lint.yml

**Comment by geoffwoollard** [comment]
You can also consider adding a docstring, e.g. "Linear Simulator." See here: https://github.com/compSPI/simSPI/blob/master/simSPI/__init__.py

**Comment by geoffwoollard** [comment]
Also @jedyeo , FYI you can do PRs to a dev of simSPI, and install from this branch. You just modify the `environment.yaml` to have some :branch in the right place. So don't feel tied in to waiting for these PRs to be on master.

**Comment by geoffwoollard** [comment]
@ninamiolane ready to merge. Important for hackathon.

**Review by ninamiolane — APPROVED** [review]
Many thanks!

**Pull request merged**